airflow/config_templates/config.yml

```
@@ -2246,6 +2246,13 @@
       type: string
       example: ~
       default: "offset"
+    - name: index_patterns
+      description: |
+        Comma separated list of index patterns to use when searching for logs (default: `_all`).
+      version_added: 2.3.3
```

```suggestion
      version_added: 2.4.0
```

Oh, and this will end up being in core 2.4.0, as this isn't a bug (though I'm glad to see it supported 🍺).